platform/pulp/app/viewsets/catalog.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import pagination
+
+from pulp.app.models import DownloadCatalog
+from pulp.app.serializers import DownloadCatalogSerializer
+from pulp.app.viewsets import NamedModelViewSet
+
+
+class DownloadCatalogPagination(pagination.CursorPagination):
+    """
+    Download catalog paginator, orders download catalog by importer when paginating.
+    """
+    ordering = 'importer'
+
+
+class DownloadCatalogViewSet(NamedModelViewSet):
+    endpoint_name = 'downloadcatalog'
```

Comment:
s/downloadcatalog/downloadcatalogs/ 
`Endpoint names (the endpoint_name attribute) should plural, not singular` [[0]](http://docs.pulpproject.org/en/3.0/nightly/contributing/3.0-development/rest-api.html#introduction)